src/bootstrap/test.rs

```diff
@@ -550,16 +550,45 @@ impl Step for Clippy {
             .stage_out(compiler, Mode::ToolRustc)
             .join(&self.host.triple)
             .join(builder.cargo_dir());
-        cargo.env("HOST_LIBS", host_libs);
-        cargo.env("TARGET_LIBS", target_libs);
+        cargo.env("HOST_LIBS", &host_libs);
+        cargo.env("TARGET_LIBS", &target_libs);
         // clippy tests need to find the driver
-        cargo.env("CLIPPY_DRIVER_PATH", clippy);
+        cargo.env("CLIPPY_DRIVER_PATH", &clippy);
 
         cargo.arg("--").args(builder.config.cmd.test_args());
 
         cargo.add_rustc_lib_path(builder, compiler);
 
-        builder.run(&mut cargo.into());
+        if builder.config.cmd.bless() {
+            // We always need to run the second command here
+            builder.info("running Clippy tests and bless");
+            try_run(builder, &mut cargo.into());
```

Comment:
This will stop the build if the cargo command returns a non-zero exit code, is there any way to continue after this?